Functions.Templates/Templates/HttpTrigger-CSharp/metadata.json

```diff
@@ -1,12 +1,13 @@
 {
     "defaultFunctionName": "HttpTriggerCSharp",
-    "description": "$HttpTriggerCSharp_description",
-    "name": "HttpTrigger",
+    "description": "$HttpTrigger_description",
+    "name": "Http trigger",
```

Comment:
It's `Http` here and `HTTP` else where. Should try to be consistent.

Reply:
Updating to HTTP